metadata.yaml

```diff
@@ -6,8 +6,9 @@ description: |
   CoreDNS provides DNS resolution for Kubernetes.
 tags:
   - networking
-series:
-  - kubernetes
+containers:
+  coredns:
+    resource: coredns-image
```

Comment:
ok, there's a new resource here.  The resulting charm we build needs to have the resource published to charmhub 

See [jenkins build-charm](https://github.com/charmed-kubernetes/jenkins/blob/master/jobs/includes/charm-support-matrix.inc#L175-L180) to add a 

`build-resources` script so the charm pushes the right oci-image to charmhub on build. 

MAYBE this `upstream-source` takes care of things here -- I don't know